python/lsst/pipe/tasks/extended_psf.py

```diff
@@ -482,10 +520,24 @@ def select_detector_refs(self, ref_list):
             region_ref_list[region_name].append(dataset_handle)
         return region_ref_list
 
+    def _find_detectors(self, bss_ref_list):
```

Comment:
Added the `@staticmethod` decorator and removed `self` parameter. However, I'm not sure we need this method any longer, and it may be a vestigial artefact from our development phase on this ticket. Keen to hear from @bazkiaei on this.

Reply:
Yes, you are right @leeskelvin . This method was created to generate a region including all existing detectors as the default region when the user do not provide regions, but since we changed our approach for that we do not need this anymore and we can remove this.